index.js

```diff
@@ -0,0 +1,86 @@
+import Book from './modules/addBook.js';
+
+let removeBtn = document.querySelectorAll('.remove');
+
+function addEvents() {
+  removeBtn.forEach((element) => {
+    element.addEventListener('click', (e) => {
+      const ref = e.target.id;
+      Book.removeBook(ref);
+      e.target.parentElement.remove();
+    });
+  });
+}
```

Comment:
- [ ] It would be better if you use arrow functions instead of the keyword function. Kindly fix this minor issue in all the functions you created. 
😄 👍